<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRulerSlider</class>
 <widget class="QWidget" name="frmRulerSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="RulerSlider" name="rulerSlider1" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>482</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
  <widget class="RulerSlider" name="rulerSlider2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>105</y>
     <width>482</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
  <widget class="RulerSlider" name="rulerSlider3" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>201</y>
     <width>482</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RulerSlider</class>
   <extends>QWidget</extends>
   <header>rulerslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
